learn/data_backup/dumps.mdx

```diff
@@ -29,11 +37,11 @@ The above code triggers a dump creation process. It also returns a [summarized t
 }
 ```
 
-In the below command, replace `1` with the `taskUid` returned by the previous command.
+The dump creation process is an asynchronous task that takes time proportional to the size of your dataset. Replace `1` with the `taskUid` returned by the previous command:
```

Comment:
```suggestion
The dump creation process is an asynchronous task that takes time proportional to the size of your database. Replace `1` with the `taskUid` returned by the previous command:
```

```suggestion
The dump creation process is an asynchronous task that takes time proportional to the size of your indexes. Replace `1` with the `taskUid` returned by the previous command:
```

Just thinking out loud. Since you can have multiple indexes with various datasets, maybe we should be more general on the term?